python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -308,6 +308,47 @@ def my_asset(context: AssetExecutionContext):
         """
         return self._step_execution_context.partition_key
 
+    @public
+    @property
+    def partition_keys(self) -> Sequence[str]:
+        """Returns a list of the partition keys for the current run.
+
+        If you want to write your asset to support running a backfill of several partitions in a single run,
+        you can use ``partition_keys`` to get all of the partitions being materialized
+        by the backfill.
+
+        Raises an error if the asset is a multi-asset. In a multi-asset, use ``asset_partition_keys_for_output``
```

Comment:
`context.partition_key_range` and `context.partition_time_window` work fine for multi-assets. Can we make this work for multi-assets as well?